src/images/zookeeper.rs

```diff
@@ -0,0 +1,90 @@
+use crate::core::Port;
+use crate::{Container, Docker, Image, WaitForMessage};
+use std::collections::HashMap;
+
+const CONTAINER_IDENTIFIER: &str = "zookeeper";
+const DEFAULT_TAG: &str = "3.5.8";
+const DEFAULT_PORT: u16 = 2181;
+
+#[derive(Debug, Default, Clone)]
+pub struct ZookeeperArgs;
+impl IntoIterator for ZookeeperArgs {
+    type Item = String;
+    type IntoIter = ::std::vec::IntoIter<String>;
+
+    fn into_iter(self) -> <Self as IntoIterator>::IntoIter {
+        vec![].into_iter()
+    }
+}
+
+#[derive(Debug)]
+pub struct Zookeeper {
+    tag: String,
+    arguments: ZookeeperArgs,
+    ports: Option<Vec<Port>>,
+}
+
+impl Default for Zookeeper {
+    fn default() -> Self {
+        Zookeeper {
+            tag: DEFAULT_TAG.to_string(),
+            arguments: ZookeeperArgs {},
+            ports: Some(vec![Port {
```

Comment:
So actually, we don't even need this! Sorry to send you on a wild goose chase here :sweat_smile: 

Looking at the source of the docker image (https://hub.docker.com/layers/zookeeper/library/zookeeper/3.6.2/images/sha256-a6ba8cd94070ecd6d30f0c2c0911dc04c7d6c7eef77e35460d8e43cdb7a0bd33?context=explore), it automatically exposes the port `2181`.

That combined with [this](https://github.com/testcontainers/testcontainers-rs/blob/3d80acb27677a5897a5a07a8d5276024a8979d93/src/clients/cli.rs#L101-L109) functionality in testcontainers makes things "just work" if people are using `Container#get_host_port`.

If we specify the port here, we force docker to use 2181 on the host as well which means we can only have one of those images running at the same time.

In conclusion, we should actually not default this here. I guess we still want to keep the `with_mapped_port` functionality but I think that would actually fit better onto the [`RunArgs`](https://github.com/testcontainers/testcontainers-rs/blob/3d80acb27677a5897a5a07a8d5276024a8979d93/src/core/docker.rs#L22) that we introduced recently.

Reply:
yeah, I got it, thanks.